ci/firebase_testlab.py

```diff
@@ -11,7 +11,8 @@
 import subprocess
 import sys
 
-BUCKET = 'gs://flutter_firebase_testlab'
+BUCKET = os.environ['STORAGE_BUCKET']
+PROJECT = os.environ['GCP_PROJECT']
```

Comment:
These should either get defaulted to something sensible or there should be an error thrown if they're not set.

Reply:
Done.